tests/monotouch-test/Foundation/CalendarTest.cs

```diff
@@ -8,11 +8,15 @@
 using SizeF=CoreGraphics.CGSize;
 using PointF=CoreGraphics.CGPoint;
 
+#if NET
+using PlatformException=ObjCRuntime.ObjCException;
+#else
 #if MONOMAC
 using PlatformException = Foundation.ObjCException;
 #else
 using PlatformException = Foundation.MonoTouchException;
 #endif
+#endif
```

Comment:
```suggestion
#if NET
using PlatformException=ObjCRuntime.ObjCException;
#elif MONOMAC
using PlatformException = Foundation.ObjCException;
#else
using PlatformException = Foundation.MonoTouchException;
#endif
```
Just my opinion, but I think this is easier to read